superset/models/core.py

```diff
@@ -763,6 +764,9 @@ def get_sqla_engine(self, schema=None, nullpool=True, user_name=None):
         if configuration:
             params['connect_args'] = {'configuration': configuration}
 
+        if('connect_args' in params):
+            update_connect_args(url,params['connect_args'])
```

Comment:
can we use objects `update` method here and then calling the method `update_connect_args` inside the update method, just as being done in `def testconn(self):` at line number 1699 for configuration object?